src/test/java/com/yugabyte/cdcsdk/testing/PostgresSinkConsumerIT.java

```diff
@@ -96,9 +90,40 @@ public static void afterClass() throws Exception {
         ybHelper.execute(UtilStrings.getDropTableStmt(DEFAULT_TABLE_NAME));
     }
 
-    @Test
+    @ParameterizedTest
+    @ValueSource(strings = {"CDCSDK" ,"Kafka connect"})
```

Comment:
Instead of passing strings directly, use ENUMs. Maybe create a java file just for `RunnerParameter.java` and then use value sources here like `RunnerParameter.CDCSDK, RunnerParameter.Connect`

Reply:
This way you will not need to worry about the values if someone even modifies it accidentally.

Reply:
I have moved this strings to UtilStrings class and now using them as value source